selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/SeleniumWebDriverHelper.java

```diff
@@ -156,45 +156,37 @@ public WebElement waitVisibility(WebElement webElement) {
    *
    * @param elementLocator
    * @param timeout waiting time in seconds
-   * @return {@code true} - if not visible,
-   *     <p>throws {@link org.openqa.selenium.TimeoutException} - if visible after timeout.
    */
-  public boolean waitInvisibility(By elementLocator, int timeout) {
-    return webDriverWaitFactory.get(timeout).until(invisibilityOfElementLocated(elementLocator));
+  public void waitInvisibility(By elementLocator, int timeout) {
+    webDriverWaitFactory.get(timeout).until(invisibilityOfElementLocated(elementLocator));
   }
 
   /**
    * Waits until {@link WebElement} with provided locator {@link By} be invisible
    *
    * @param elementLocator
-   * @return {@code true} - if not visible,
-   *     <p>throws {@link org.openqa.selenium.TimeoutException} - if visible after timeout.
    */
-  public boolean waitInvisibility(By elementLocator) {
-    return waitInvisibility(elementLocator, DEFAULT_TIMEOUT);
+  public void waitInvisibility(By elementLocator) {
+    waitInvisibility(elementLocator, DEFAULT_TIMEOUT);
   }
 
   /**
    * Waits until provided {@link WebElement} be invisible
    *
    * @param webElement
    * @param timeout waiting time in seconds
-   * @return {@code true} - if not visible,
-   *     <p>throws {@link org.openqa.selenium.TimeoutException} - if visible after timeout.
```

Comment:
`@throw {@link org.openqa.selenium.TimeoutException} - if is visible during timeout.` could be still useful for reader.